packages/react-table/src/components/DataTable.tsx

```diff
@@ -96,7 +100,16 @@ const DataTable = <T extends Record<string, any>>({
                 >
                   {headerGroup.headers.map(column => (
                     <div
-                      {...column.getHeaderProps()}
+                      {...column.getHeaderProps({
+                        className: classNames(
+                          (
+                            column as unknown as { className?: string }
+                          )?.className,
+                          (
+                            column as unknown as { headerClassName?: string}
+                          )?.headerClassName
```

Comment:
```suggestion
                          column?.className, column?.headerClassName
```

这样不行吗？是 typescript 报错吗？

Reply:
typescript 会报错，查看了下 Column 的范型也没什么地方可以加入扩展，官方 issue 也找不到相关的写法